src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -78,20 +87,20 @@ const mongoSchemaFromFieldsAndClassNameAndCLP = (fields, className, classLevelPe
 }
 
 
-export class MongoStorageAdapter {
+export class MongoStorageAdapter implements StorageAdapter, IndexingStorageAdapter {
   // Private
   _uri: string;
   _collectionPrefix: string;
   _mongoOptions: Object;
   // Public
   connectionPromise;
```

Comment:
Is it possible to add a type here or not worthwhile?

Reply:
probably yes,